service/iot-service-client/pom.xml

```diff
@@ -86,18 +86,6 @@
             <version>1.18.8</version>
             <scope>provided</scope>
         </dependency>
-        <dependency>
-            <groupId>com.microsoft.azure</groupId>
-            <artifactId>azure-storage</artifactId>
```

Comment:
Even though this was scoped as a test dependency, we weren't actually using this in the service client anywhere, so I removed it